tests/ci/profile_handler.go

```diff
@@ -331,7 +398,19 @@ func GenerateClusterCreationArgsByProfile(token string, profile *Profile) (clust
 
 	if profile.STS {
 		majorVersion := GetMajorVersion(profile.Version)
-		accountRolesOutput, err := PrepareAccountRoles(token, clusterArgs.ClusterName, profile.UnifiedAccRolesPath, clusterArgs.AWSRegion, majorVersion, profile.ChannelGroup, profile.GetClusterType())
+		var accountRolesOutput *EXE.AccountRolesOutput
+
+		shared_vpc_role_arn := ""
+		if profile.SharedVpc {
+			// FIXME:
+			//	To create Shared-VPC compatible policies, we need to pass a role arn to create_account_roles module.
+			//  But we got an chicken-egg prolems here:
+			//		* The Shared-VPC compatible policie requries installer role
+			//		* The install role (account roles) require Shared-VPC ARN.
+			//  Use hardcode as a temporary solution.
+			shared_vpc_role_arn = fmt.Sprintf("arn:aws:iam::641733028092:role/%s-shared-vpc-role", clusterArgs.ClusterName)
```

Comment:
I would like to put this part to the manifest file rather than in go code.

Reply:
> manifest file

Do you mean the account role TF manifest file?  The module creates different policies based on the value of this role arn.

Reply:
OK, I am good with it now.